addons/ai/functions/fnc_unpackStaticWeaponPFH.sqf

```diff
@@ -0,0 +1,74 @@
+#include "script_component.hpp"
+/*
+ * Author: Ampersand
+ * Move units together if needed and unpacks a static weapon from units' backpacks.
+ *
+ * Arguments:
+ * 0: Gunner <OBJECT>
+ * 1: Assistant <OBJECT>
+ * 2: Target Position <ARRAY>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [_gunner, _assistant] call zen_ai_fnc_unpackStaticWeaponPFH
+ *
+ * Public: No
+ */
+
+#define DISTANCE_CLOSE 3
+#define MOVE_DELAY 5
+#define MOVE_TIMEOUT 60
+
+params ["_gunner", "_assistant", ["_targetPos", [], [[]], 3]];
+
+// Order assistant to move to gunner and start pfh
+_gunner setVariable [QGVAR(statePATH), _gunner checkAIFeature "PATH"];
+_gunner disableAI "PATH";
+[_assistant] joinSilent grpNull;
+private _group = group _assistant;
+_group setBehaviourStrong "CARELESS";
+_group deleteGroupWhenEmpty true;
+_group enableAttack false;
```

Comment:
Can probably use new `setCombatBehaviour` and `setUnitCombatMode` commands to avoid affecting the entire group. Also, likely make units more responsive even in combat.